test/property_test.c

```diff
@@ -198,6 +199,9 @@ static const struct {
     { 1, "a=2, n=012345678" },  /* Bad octal digit */
     { 0, "n=0x28FG, a=3" },     /* Bad hex digit */
     { 0, "n=145d, a=2" },       /* Bad decimal digit */
+    { 0, "n=0xffffffffffffffff, a=3" },      /* Hex overflow */
+    { 0, "n=9223372037000000000d, a=2" },    /* Decimal overflow */
```

Comment:
Ideally for these tests we'd have positive (should pass) tests for a value of `INT64_MAX` in dec, hex and octal, as well as negative (should fail) tests for a value of `INT64_MAX + 1` in dec, hex and octal.

Reply:
modified tests such that negative tests use INT64_MAX+1 in decimal, hex and octal, and added positive tests for hex/decimal/octal using INT64_MAX